File: property_managments/property_company/override/territory.py
import frappe
from erpnext.setup.doctype.territory.territory import Territory

class CustomTerritory(Territory):

    def save(self, *args,**kwargs):
        print(888888888888)
        super().save(*args, **kwargs)
    pass



    def validate(self):
      if self.terrot== "Country" and self.is_group==0:
          frappe.throw(" Country can not be is_group 1")
      super().validate()

File: property_managments/public/js/territory.js
frappe.ui.form.on("Territory", {
	refresh: function(frm) {
		console.log("1234567890");

		},
        terrot:function(frm){
            if(frm.doc.terrot=="Country"
            || frm.doc.terrot=="City" ||
            frm.doc.terrot=="Town"
            ){
               frm.set_value("is_group", 1);
            }
        else{
             frm.set_value("is_group", 0);
             
        }
        }
});
